android/src/main/kotlin/io/customer/customer_io/CustomerIoPlugin.kt

```diff
@@ -142,9 +154,28 @@ class CustomerIoPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
         }
     }
 
+    private fun registerDeviceToken(params: Map<String, Any>) {
```

Comment:
Mostly for consistency, Since other methods are sending `Map` we have now a utility method that verifies the map and its keys. 

Also, if we ever wanted to send more params along with `token` we won't have to change the signature of the method or implementation but rather just add another `pair` to the map. 

But I agree that this looks like overkill and I might deviate away from this as well if we get 1 more method that just takes 1 `string` parameter.